edn/Cargo.toml

```diff
@@ -13,6 +13,8 @@ readme = "./README.md"
 [dependencies]
 num = "0.1.35"
 ordered-float = "0.4.0"
+regex = "0.2.1"
```

Comment:
It's not critical, but checking for `.startswith("?")` should be enough; no need for regexes right away.